build.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencies {
 
```

Comment:
Got a ticker on the board to start migrating to the latest dps gradle plugin.

Reply:
Just to 1.1.2 for now - there are some issues with v2.0.0